dla_cnn/desi/uniform_zdla_nhi.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import numpy as np
+import random
+from dla_cnn.desi.load_fits_files import load_DesiMocks,sightline_retriever
+from dla_cnn.desi.uniform_snr import split_bins
+
+
+# In[2]:
+
+
+def uniform_z_nhi(idarray,z_bins,nhi_bins,num,Mocks):
```

Comment:
move this method into `training_sets.py`

And you should be using the Summary table from @zoujiaqi99 
not re-building the `Sightline` objects